### ISSUE TITLE
[Term Entry] PyTorch nn Loss Functions

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#5233]
### Reason/inspiration (optional)

We would like a new entry on the `Loss Functions` in PyTorch. The entry should go in a new file under `content/pytorch/concepts/nn/terms/loss-functions/loss-functions.md`.      _x000D_
  _x000D_
The entry should include:  _x000D_
  _x000D_
- An introduction to the concept     _x000D_
  _x000D_
- A `Syntax` section that provides the syntax for the concept (if applicable)     _x000D_
  _x000D_
- An `Example` section that provides an example demonstrating the concept in use (if applicable)     _x000D_
  _x000D_
Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by saldanhad** [comment]
Hi @codecademy-docs, could you clarify whether you're looking for a general overview of how loss functions are used in PyTorch, or do you want the entry to focus specifically on certain loss functions like `MSELoss`or `CrossEntropyLoss`?

**Comment by mamtawardhani** [comment]
Hey @avdhoottt @Sriparno08 can you help here?

**Comment by Sriparno08** [comment]
Hey @mamtawardhani, I think including a general overview of how loss functions are used plus a brief explanation for all the loss functions in PyTorch will be good.

What's your take, @avdhoottt?

**Comment by mamtawardhani** [comment]
> Hey @mamtawardhani, I think including a general overview of how loss functions are used plus a brief explanation to all the loss functions in PyTorch will be good.
> 
> What's your take, @avdhoottt?

@deenovita here is the requirement, if you are still interested, you can let us know 😄. Thank you!
Thank you @Sriparno08 😃

**Comment by danitellini** [comment]
Can I please be assigned this issue? @mamtawardhani

**Comment by mamtawardhani** [comment]
Hey @danitellini You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

**Comment by Radhika-okhade** [comment]
@danitellini are you working on this?

**Comment by danitellini** [comment]
Yes
